### PR TITLE
Explicitly set character encoding

### PR DESCRIPTION
**Pull request by chooban**
There are some comments in the code which are written using the
specified character set. Without it, maven defaults to UTF-8 and
fails to compile the source files.

### CODE REVIEW

**Comment by erikzhang** [comment]
What about changing the source files to UTF-8?

**Comment by chooban** [comment]
That would be more sensible, so long as we can ensure everyone's on the same page. I presume it's possible to set that explicitly in Eclipse and have it as part of the .project file.

File: .settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (erikzhang) [inline .settings/org.eclipse.core.resources.prefs]: After changing the encoding of sources, is this file necessary?

Comment (chooban) [reply .settings/org.eclipse.core.resources.prefs]: My understanding is that this will make new files created in Eclipse use UTF-8 encoding, otherwise the default is to use the system encoding which could result in the issue appearing again. I'm not sure how to go about testing that theory, unfortunately.

**Comment by peterszatmary** [comment]
Encoding should be set rather in pom.xml file than in eclipse metafiles. Because they are eclipse specific. Dont forget people they are using other IDEs for example Intelijj Idea or Netbeans ...

My recomendation is to set it like maven properties and a;so in maven compiler plugin.

`<properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
  </properties>

  <build>
    <sourceDirectory>src</sourceDirectory>
    <plugins>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.6.1</version>
        <configuration>
          <source>1.8</source>
          <target>1.8</target>
          <encoding>${project.build.sourceEncoding}</encoding>
        </configuration>
      </plugin>
    </plugins>
  </build>`

**Comment by peterszatmary** [comment]
The failing compilation is now solving in this pull request by translating chinese to english comments. See please [#4](https://github.com/neo-project/neo-devpack-java/pull/4). ALso BOM was deleted. After that we can add explicit UTF-8 maven configuration if you like.
